src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -221,6 +231,15 @@ public boolean isDependentArgumentAllowed(final Class<?> dependentClass) {
      */
     @Override
     public void validateArguments() {
+        // throw if any filter does not exists
+        final Set<String> nonExistant = new HashSet<>(disableFilters);
+        nonExistant.removeAll(readFilterNames);
+        if (!nonExistant.isEmpty()) {
+            throw new CommandLineException.BadArgumentValue(
+                    String.format("The read filter(s) specified do not exists: ",
+                            Utils.join(", ", nonExistant)));
+        }
```

Comment:
True, it is much cleaner than this implementation. I've done this to point out all the non-existent in the error, as a good way to remove in the same batch all possible invalid filters. Anyway, I simplified as you suggested and changed the error message.